tools/wheel_resolver/test/BUILD

```diff
@@ -6,8 +6,8 @@ python_test(
 
 gentest(
     name = "e2e_test",
-    data = ["//tools/wheel_resolver"],
+    data = ["//tools/wheel_resolver/src:resolve"],
     no_test_output = True,
-    test_cmd = "PKG=tensorflow; OUTS=$PKG.whl; export OUTS; $DATA --package \"$PKG\"; \
+    test_cmd = "PKG=cryptography; OUTS=$PKG.whl; export OUTS; $DATA --package \"$PKG\"; \
```

Comment:
Wheel resolver tool now fails to download tensorflow. It also fails to download a subset of the things in third_party/python - I'll raise a separate issue for this.